ibis/backends/dask/aggcontext.py

```diff
@@ -0,0 +1,234 @@
+import operator
+from typing import Any, Callable, Dict, Tuple, Union
+
+import dask.dataframe as dd
+from dask.dataframe.groupby import SeriesGroupBy
+
+import ibis
+import ibis.expr.datatypes as dt
+from ibis.backends.pandas.aggcontext import (
+    AggregationContext,
+    compute_window_spec,
+    construct_time_context_aware_series,
+    get_time_col,
+    window_agg_udf,
+    wrap_for_agg,
+)
+
+# TODO Consolidate this logic with the pandas aggcontext.
+# This file is almost a direct port of the pandas aggcontext.
+
+
+class Summarize(AggregationContext):
+    __slots__ = ()
+
+    def agg(self, grouped_data, function, *args, **kwargs):
+        if isinstance(function, str):
+            return getattr(grouped_data, function)(*args, **kwargs)
+
+        if not callable(function):
+            raise TypeError(f'Object {function} is not callable or a string')
```

Comment:
tests hit this?

Reply:
yeah this gets hit by `test_aggregation.test_reduction_ops`